### ISSUE TITLE
[Doc]: Как отозвать токен или выйти из приложения?

### ISSUE THREAD

**Issue by Marcus-Rise** [VKCOM/vkid-web-sdk#13]
### Описание

Не нашел функционала, как отозвать токен или описания функционала выхода из vkid

**Comment by ByQwert** [comment]
https://id.vk.com/about/business/go/docs/ru/vkid/archive/1.60/vk-id/logout ?

**Comment by Marcus-Rise** [comment]
> https://id.vk.com/about/business/go/docs/ru/vkid/archive/1.60/vk-id/logout ?

Да, видел этот код, но не нашел его исходный код реализации в репозитории 

```js
Config.init({
appId: 0, // Подставьте идентификатор вашего приложения.
superAppTokenV2: '...' // Подставьте SuperApp token. 
});
Connect.logout().then(() => {
// Можно добавить свой обработчик.
});
```


Какой рест апи испольлзуется в этом случае?

**Comment by ByQwert** [comment]
Как вариант, можно посмотреть, как тут работает логаут:
https://id.vk.com/about/business/demo/#/

```
https://id.vk.com/logout?origin=https%3A%2F%2Fid.vk.com&host_app_id=***&superapp_token=***
```

**Comment by levtsypanov** [comment]
> > https://id.vk.com/about/business/go/docs/ru/vkid/archive/1.60/vk-id/logout ?
> 
> Да, видел этот код в разделе документации «Архив», но не нашел его исходную реализацию в репозиториях. Упоминания модуля `Connect`также не встречались.
> 
> ```js
> Config.init({
> appId: 0, // Подставьте идентификатор вашего приложения.
> superAppTokenV2: '...' // Подставьте SuperApp token. 
> });
> Connect.logout().then(() => {
> // Можно добавить свой обработчик.
> });
> ```
> 
> Какой рест апи используется в этом случае?

Тоже сталнулся с этой проблемкой. Реализация логаута через Connect, из старой библиотеки [VK SuperAppKit](https://www.npmjs.com/package/@vkontakte/superappkit). Надеюсь в ближайшее время добавят. Пока воспользуюсь вариантом предложенным [ByQwert](https://github.com/ByQwert)

**Comment by Thisman** [comment]
> Надеюсь в ближайшее время добавят.

Привет! Мы сейчас готовим реализацию OAuth2.1 в SDK, где в том числе добавим метод logout для инвалидации токена
